action.yml

```diff
@@ -58,6 +58,15 @@ inputs:
     description: "Upload artifact to workflow"
     default: "true"
 
+  upload-artifact-retention-days:
```

Comment:
The `retentionDays` is optional for `uploadArtifact`, but i can make input with default 0.
It will just always set this parameter even if its optional on TS code.

Reply:
Updated